src/__swaps__/screens/Swap/Swap.tsx

```diff
@@ -208,7 +208,10 @@ const ExchangeRateBubbleAndWarning = () => {
 
 export const styles = StyleSheet.create({
   rootViewBackground: {
-    borderRadius: IS_ANDROID ? 20 : ScreenCornerRadius,
+    borderTopLeftRadius: IS_ANDROID ? 20 : ScreenCornerRadius,
+    borderTopRightRadius: IS_ANDROID ? 20 : ScreenCornerRadius,
+    borderBottomLeftRadius: IS_ANDROID ? 0 : ScreenCornerRadius,
+    borderBottomRightRadius: IS_ANDROID ? 0 : ScreenCornerRadius,
```

Comment:
Noticed on Android that the bottom of swaps screen is rounded and looks a little weird.